### DISCUSSION TITLE
RESTAPI

### DISCUSSION THREAD

**Discussion by dikshauat** [Sunbird-Ed/Community#201] [General Discussion]
@keshavprasadms , our keycloak is working fine. Checked its configuration and found no issues with it. We were able to generate the required access token on postman but currently we are not able to create any user through RESTAPI, Please check the below snapshot and kindly assist us
![image](https://user-images.githubusercontent.com/112255486/193008334-f5390eca-5201-43d9-9b0e-78f6230197de.png)

**Comment by beepdot** [comment]
Please post logs from learner service.

**Reply by dikshauat** [reply]
@keshavprasadms , we tried resolving the issue by uninstalling Apimanager on kubectl . Now when we tried deploying it again, we are getting the following error 

![image](https://user-images.githubusercontent.com/112255486/193061249-2eee6c6d-e10f-4a3e-afda-d2a0952917b5.png)

**Reply by dikshauat** [reply]
@keshavprasadms , I think when the apimanager pod will come up again , the RESTAPI issue will automatically resolve. Kindly help me resolving the issue .

**Reply by beepdot** [reply]
Please post logs from apimanager pods.

**Reply by dikshauat** [reply]
@keshavprasadms , here are the required logs
![image](https://user-images.githubusercontent.com/112255486/193238704-af70e524-3eb8-46fd-9bb1-805956a54a0d.png)

**Reply by beepdot** [reply]
What is the version of kong running?
